src/Database/Sql.php

```diff
@@ -896,6 +902,10 @@ public function valueSet(string $table, $values, string $separator = ',', bool $
         foreach ($values as $column => $value) {
             $key = $this->columnName($table, $column, $enforceQualified);
 
+            if ($key === null) {
+                continue;
+            }
+
             if (in_array($value, static::$literals, true) === true) {
                 $query[] = $key . ' = ' . ($value ?: 'null');
```

Comment:
I have a question independent of this issue. Shouldn't `x IS NULL` instead of `x = null`?

Reply:
Could be, yes. I think both should be equivalent, right? The current code is pretty short.

Reply:
I can create a separate PR for this bug if you want.

Reply:
By the way, I guess this is valid for the `where` condition.

Reply:
I will 👍